### ISSUE TITLE
Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name 20161226_135947.jpg could not be located

### ISSUE THREAD

**Issue by meras1894** [nextcloud/android#475]
### Actual behaviour
I want to upload a picture out of the gallery using the share-functionality. Upload is not possible, I get an unknown error. I tried that at my own nextcloud-installation and at an other nextcloud-installation (also NC11) at pixelx.de

What does work:
- Auto-Upload
- Manual Upload directly out of the app

### Expected behaviour
Picture should get uploaded.
 
### Steps to reproduce
1. Going to gallery
2. Select a picture
3. Selecting Share-Button
4. Selecting Nextcloud
5. Selecting Upload-Button

### Environment data
Android version: 6.0.1
Device model: Samsung Galaxy S7
Stock or customized system: Stock
Nextcloud app version: 1.4.0
Nextcloud server version: Nextcloud 11

#### Web server error log
```
There is no error log on this action.
```

#### Nextcloud log (data/nextcloud.log)
```
	Sabre\DAV\Exception\NotFound: HTTP/1.1 404 File with name 20161226_135947.jpg could not be located

0.    /var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php - line 81: OCA\DAV\Connector\Sabre\ObjectTree->getNodeForPath('20161226_135947...')
1.    [internal function] Sabre\DAV\CorePlugin->httpGet(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))
2.    /var/www/nextcloud/3rdparty/sabre/event/lib/EventEmitterTrait.php - line 105: call_user_func_array(Array, Array)
3.    /var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 479: Sabre\Event\EventEmitter->emit('method GET', Array)
4.    /var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/CorePlugin.php - line 253: Sabre\DAV\Server->invokeMethod(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response), false)
5.    [internal function] Sabre\DAV\CorePlugin->httpHead(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))
6.    /var/www/nextcloud/3rdparty/sabre/event/lib/EventEmitterTrait.php - line 105: call_user_func_array(Array, Array)
7.    /var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 479: Sabre\Event\EventEmitter->emit('method HEAD', Array)
8.    /var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 254: Sabre\DAV\Server->invokeMethod(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))
9.    /var/www/nextcloud/apps/dav/appinfo/v1/webdav.php - line 60: Sabre\DAV\Server->exec()
    /var/www/nextcloud/remote.php - line 165: require_once('/var/www/nextcl...')
10.    {main}
```

**Comment by tobiasKaminsky** [comment]
@meras1894 thank you for taking the time to report your problem.
Is this still valid in the latest version?
Is the shared image somehow "restricted", e.g. not available by other apps?

**Comment by meras1894** [comment]
@tobiasKaminsky Sorry for the late response. I have been on holiday^^

The problem ist still valid.
So, sometimes it's working, but most of the time, it doesn't work.
I try it with the pictures, made with the camera. So I don't think, they are "restricted"(?).
I tested it in Wifi, I tried it in mobile data. Sometimes working, sometimes not.
I tested it out of the galery app, I tested it out of the opened image. No influence on the result. Sometimes it's working, sometimes not.

I don't understand, why it's working some time, and some time doesn't.
